src/varlink.js

```diff
@@ -53,15 +53,23 @@ function connect(address) {
 
         for (let i = 0; i < chunks.length; i += 1) {
             const message = JSON.parse(chunks[i]);
-
-            let { resolve, reject } = pending.shift();
-            if (message.error)
-                reject({
-                    error: message.error,
-                    parameters: message.parameters
-                });
-            else
-                resolve(message.parameters);
+            const { resolve, reject, eventCallback } = pending.shift();
+
+            if (eventCallback) {
+                if (message.parameters && message.parameters.events) {
```

Comment:
`message.parameters.events` is specific to podman's API, isn't it? I'd prefer if `varlink.js` stays a generic varlink client implementation so that we can split it out at some point.

Reply:
The error case is not handled for `more` calls.

Reply:
Done, thanks for the catch